clintrials/core/numerics.py

```
@@ -123,27 +131,27 @@ def adaptive_mc_integration(
+    hi: float,
     *,
-    method="grid",
-    n_points=2001,
-    adaptive_limits=True,
-    edge_frac=0.02,
-    tail_mass_tol=1e-3,
-    expand_factor=1.0,
-    max_expansions=6,
-    warn_on_max=True,
-    return_diagnostics=False,
+    method: str = "grid",
```

Ruff (0.15.21)

[warning] Unused function argument: `method`

(ARG001)